okapi-core/src/main/java/org/folio/okapi/service/impl/PostgresHandle.java

```diff
@@ -63,6 +68,15 @@
         Config.getSysConf("postgres_user", "okapi", new JsonObject()), conf));
     connectOptions.setPassword(Config.getSysConf("postgres_password", "okapi25", conf));
     connectOptions.setDatabase(Config.getSysConf("postgres_database", "okapi", conf));
+    String serverPem = Config.getSysConf("postgres_server_pem", null, conf);
+    if (serverPem != null) {
+      connectOptions.setSslMode(SslMode.VERIFY_FULL);
+      connectOptions.setHostnameVerificationAlgorithm("HTTPS");
+      connectOptions.setPemTrustOptions(
+          new PemTrustOptions().addCertValue(Buffer.buffer(serverPem)));
```

Comment:
Okapi exits with this error message if the provided certificate is invalid:
`FATAL PostgresQuery        getCon failed Postgres Server does not handle SSL connection`

This is similar to the message when when Okapi exits because of wrong password:
`FATAL PostgresQuery        getCon failed password authentication failed for user "okapi"`